docs/context/ref/ChatMessage.md

```diff
@@ -0,0 +1,51 @@
+---
+id: ChatMessage
+sidebar_label: ChatMessage
+title: ChatMessage
+hide_title: true
+---
+# `ChatMessage`
+
+Message to send to a chat.
```

Comment:
```suggestion
A context representing a chat message. Typically used to send the message or to pre-populate a message for sending.
```

I wonder if the description should leave the door open to using this to also embody messages received, e.g. if you were able to subscribe to (filtered?) messages in a room. This could be built out via an intent that returned a `PrivateChannel`.

Reply:
This should now be resolved by the latest commit.